examples/mesh_stereo.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import depthai as dai
+import argparse
+
+"""
+If one or more of the additional depth modes (lrcheck, extended, subpixel)
+are enabled, then:
+ - depth output is FP16. TODO enable U16.
+ - median filtering is disabled on device. TODO enable.
+ - with subpixel, either depth or disparity has valid data.
+
+Otherwise, depth output is U16 (mm) and median is functional.
+But like on Gen1, either depth or disparity has valid data. TODO enable both.
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-res",
+    "--resolution",
+    type=str,
+    default="720",
+    help="Sets the resolution on mono cameras. Options: 800 | 720 | 400",
+)
+parser.add_argument(
+    "-md",
+    "--mesh_dir",
+    type=str,
+    default=None,
+    help="Output directory for mesh files. If not specified mesh files won't be saved",
+)
+parser.add_argument(
+    "-rect",
+    "--out_rectified",
+    default=False,
+    action="store_true",
+    help="Generate and display rectified streams",
+)
+parser.add_argument(
+    "-lr",
+    "--lrcheck",
+    default=False,
+    action="store_true",
+    help="Better handling for occlusions",
+)
+# NOTE: Not yet implemented
+# parser.add_argument(
+#     "-e",
+#     "--extended",
+#     default=False,
+#     action="store_true",
+#     help="Closer-in minimum depth, disparity range is doubled",
+# )
+parser.add_argument(
+    "-s",
+    "--subpixel",
+    default=False,
+    action="store_true",
+    help="Better accuracy for longer distance, fractional disparity 32-levels",
+)
+parser.add_argument(
+    "-m",
+    "--median",
+    type=str,
+    default="7x7",
+    help="Choose the size of median filtering. Options: OFF | 3x3 | 5x5 | 7x7 (default)",
+)
+parser.add_argument(
+    "-static",
+    "--static_frames",
+    default=False,
+    action="store_true",
+    help="Run stereo on static frames passed from host 'dataset' folder",
+)
+args = parser.parse_args()
+
+resolutionMap = {"800": (1280, 800), "720": (1280, 720), "400": (640, 400)}
+if args.resolution not in resolutionMap:
+    exit("Unsupported resolution!")
+
+resolution = resolutionMap[args.resolution]
+
+mesh_directory = args.mesh_dir  # Output dir for mesh files
+
+out_rectified = args.out_rectified  # Output and display rectified streams
+lrcheck = args.lrcheck  # Better handling for occlusions
+extended = False  # NOTE: Not yet implemented. Closer-in minimum depth, disparity range is doubled
```

Comment:
We do have extended. I think the note came from an old file. you can remove that enable the argument for this